src/visualEditorPage.js

```diff
@@ -8,8 +8,6 @@ define(
 			'src/debug',
 			'../src/styleUtils',
 			'../src/errorHandler',
-			'../external/downloadify/swfobject',
-			'../external/downloadify/downloadify.min',
```

Comment:
I couldn't find any calls to Downloadify in this file (or elsewhere in the repository). My guess is that it's just cruft.